diff --git a/services/fabricv4/test/api_cloud_routers_test.go b/services/fabricv4/test/api_cloud_routers_test.go
index 4221152..eaf2468 100644
--- a/services/fabricv4/test/api_cloud_routers_test.go
+++ b/services/fabricv4/test/api_cloud_routers_test.go
@@ -82,7 +82,7 @@ func Test_fabricv4_CloudRoutersApiService(t *testing.T) {
 	t.Run("Test CloudRoutersApiService GetCloudRouterPackageByCode", func(t *testing.T) {
 		t.Skip("skip test") // remove to run test
 
-		var routerPackageCode RouterPackageCode
+		var routerPackageCode openapiclient.RouterPackageCode
 
 		resp, httpRes, err := apiClient.CloudRoutersApi.GetCloudRouterPackageByCode(context.Background(), routerPackageCode).Execute()
 
