diff --git a/services/fabricv4/test/api_precision_time_test.go b/services/fabricv4/test/api_precision_time_test.go
index 2fb4344..bcfca38 100644
--- a/services/fabricv4/test/api_precision_time_test.go
+++ b/services/fabricv4/test/api_precision_time_test.go
@@ -71,7 +71,7 @@ func Test_fabricv4_PrecisionTimeApiService(t *testing.T) {
 	t.Run("Test PrecisionTimeApiService GetTimeServicesPackageByCode", func(t *testing.T) {
 		t.Skip("skip test") // remove to run test
 
-		var packageCode GetTimeServicesPackageByCodePackageCodeParameter
+		var packageCode openapiclient.GetTimeServicesPackageByCodePackageCodeParameter
 
 		resp, httpRes, err := apiClient.PrecisionTimeApi.GetTimeServicesPackageByCode(context.Background(), packageCode).Execute()
 
